pkg/ucp/credentials/types.go

```diff
@@ -34,6 +34,12 @@ const (
 
 	// AzureWorkloadIdentityCredentialKind represents the kind of Azure workload identity credential.
 	AzureWorkloadIdentityCredentialKind = ucp_dm.AzureWorkloadIdentityCredentialKind
+
+	// AWSAccessKeyCredentialKind represents the kind of AWS access key credential.
+	AWSAccessKeyCredentialKind = ucp_dm.AWSAccessKeyCredentialKind
+
+	// AWSIRSACredentialKind represents the kind of AWS IRSA credential.
+	AWSIRSACredentialKind = ucp_dm.AWSIRSACredentialKind
```

Comment:
Where are these constants used? Wondering why do we need to declare a new constant that references existing constant, instead of directly referencing the original definition.

Reply:
This can be a nice pattern because it makes the package self-contained.

Reply:
I added them to model azure, but looks like they are not needed. I removed both AWS and AZ constants that arent being currently used in our code.

Reply:
AZ constant was actually used in a test.. Removed just AWS constants that I introduced.

Reply:
> This can be a nice pattern because it makes the package self-contained. Consumers of the package can work with a single import if its done right.

@rynowak constants were being copied from another package by referencing them, which defeats the purpose of making the package self-contained (if that's the goal). Ideally, I would abstract all of them out in a separate package rather than duplicating them if we really need same constants to be referenced from different packages.

Reply:
> AZ constant was actually used in a test.. Removed just AWS constants that I introduced.

@nithyatsu could you point me to the test that is using it? Trying to understand why datamodel package can't be referenced from the test since there aren't any tests within this credentials package. Can't seem to find references from github dev editor

Reply:
/radius/pkg/azure/credential/ucpcredentials_test.go line 44
We just have defined all constants in the types.go, and instead of importing the datamodel package, imported just `sdk_cred "github.com/radius-project/radius/pkg/ucp/credentials"`